src/grafana_dashboards/node-exporter-full.json

```
@@ -23193,8 +23193,8 @@
     ]
   },
   "timezone": "browser",
-  "title": "System Resources",
+  "title": "System Resources (container)",
   "uid": "rYdddlPWk",
```

UID is overlapping with the other charm. How about:
```suggestion
  "uid": "rYdddlPWk-1",
```